app/models/group.rb

```diff
@@ -12,10 +12,10 @@
 class Group < ActiveRecord::Base
   validates_presence_of :name, :description
 
-  has_many :group_members, foreign_key: :groupid
+  has_many :group_members, foreign_key: :groupid, dependent: :destroy
   has_many :members, -> { order 'name' }, through: :group_members, source: :user
-  has_many :meetings, -> { order 'created_at DESC' }, foreign_key: :groupid
-  has_many :meetings, foreign_key: :groupid
+  has_many :meetings, -> { order 'meetings.created_at DESC' },
```

Comment:
Are you talking about the `meetings.created_at` change or the removal of the duplicate has_many :meetings association?
For the first, I'm pretty sure it was working find before this, but when I added both dependent destroys to both Group has_many :meeetings and Meeting has_many :meeting_members  and tried calling group.destroy, I got ambiguous column errors because it was trying to also join meeting_members. So technically group.members was working as expected but it wouldn't have worked if you tried additional joins. The Group#destroy unit test in this PR should catch that regression.